### PR TITLE
[BUG FIX] Update changelog for v0.9.0 release

### PR DESCRIPTION
**Pull request by LEDThereBeLight**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,153 +3,195 @@
...
-  - Use section slugs instead of ids in storage service URLs for delivery endpoints
-  - Fix a crash when an existing logged-in user accesses the Register Institution page
-  - Activity feedback fixes and unit tests
-  - Remove support for image floating to fix display issues in text editors
-  - Change activity rule, outcome modeling for use in adaptive activities
-  - Fix an issue when creating section allows multiple sections to be created
-  - Improved rendering robustness when content elements are missing key attributes
-  - Disable access to OpenAPI docs in production
+
+## 0.9.0 (2021-4-22)
```
Comment (eliknebel) [inline CHANGELOG.md:9]: Am I reading this diff right? Are there two entries for 0.9.0?

**Review by eliknebel — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (eliknebel) [reply CHANGELOG.md:9]: Since this PR is mostly a process step, we dont need an update changelog entry for it

Comment (LEDThereBeLight) [reply CHANGELOG.md:9]: there should be 1 for unreleased and 1 for 0.9.0

Comment (LEDThereBeLight) [reply CHANGELOG.md:9]: I got rid of the 0.9.1 after we talked in slack

**Review by eliknebel — APPROVED** [review]

**Pull request merged**